doc/content/getting-started/ttn/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "The Things Network"
+description: ""
+distributions: "Community"
+---
+
+{{< figure src="logo_TTN.svg" class="float plain" width="150px" >}}
+
+The Things Network is a global collaborative Internet of Things ecosystem that creates networks, devices and solutions using LoRaWAN.
+
+The Things Network runs {{% tts %}} Community, which is a crowdsourced, open and decentralized LoRaWAN network. This network is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.
+
+<!--more-->
+
+[Login or create an account](https://console.cloud.thethings.network/) to get started with The Things Network and start using {{% tts %}} Console.
```

Comment:
Maybe say something like "Choose the cluster closest to you geographically, then log in or create an account to......"

Reply:
why?